service/discovery/azure/compute_test.go

```diff
@@ -3278,6 +3278,79 @@ func Test_getSecretURI(t *testing.T) {
 	}
 }
 
+func Test_azureComputeDiscovery_getRedundancy(t *testing.T) {
+	type fields struct {
+		azureDiscovery *azureDiscovery
+	}
+	type args struct {
+		app *armappservice.Site
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		args   args
+		wantR  *voc.Redundancy
+	}{
+		{
```

Comment:
Maybe add an additional check for the missing property.

Reply:
Good point